### PR TITLE
Adb 384 qr outbound component

### PR DESCRIPTION
**Pull request by catluong101digital**
### Why

Implementing new UI and QR outbound flow and BE

Link to Jira ticket: [<place here>](https://acsdigital.atlassian.net/browse/APB-384)

### CODE REVIEW

File: src/components/available-balance-modal/index.tsx
Comment (catluong101digital) [inline src/components/available-balance-modal/index.tsx:1]: new component

File: src/components/transfer-details-component/qr-transfer-details-component.tsx
Comment (catluong101digital) [inline src/components/transfer-details-component/qr-transfer-details-component.tsx:1]: new component clone from transfer-details-component

File: src/components/available-balance-modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                },
+            }}
+            useSafeArea
+        >
+            <View style={styles.closeButtonContainer}>
+                <TouchableOpacity onPress={onClose} style={styles.closeIcon}>
+                    <CloseIcon width={8} height={8}/>
+                </TouchableOpacity>
+                <Text style={styles.titleStyle}>{'Available Balance'}</Text>
+                <Text style={styles.subtitleStyle}>{'Available balance is the account balance above the minimum account balance of RM20.'}</Text>
```
Comment (hiteshdigit101) [inline src/components/available-balance-modal/index.tsx:49]: Please use this text from translation file

File: src/components/available-balance-modal/style.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      width: 32,
+      height: 32,
+      justifyContent: 'center',
+      alignItems: 'center',
+      alignSelf: 'flex-end',
+    },
+    titleStyle: {
+      fontSize: 24,
+      fontWeight: '600',
+      fontFamily: 'Poppins',
```
Comment (hiteshdigit101) [inline src/components/available-balance-modal/style.ts:21]: Please use fonts and colors from theme-component

File: src/components/qr-transfer-component/scan-to-pay-component/index.tsx [hunk 3]
```diff
@@ -1,93 +1,158 @@
...
+    const [isLoading, setIsLoading] = useState<boolean>(false);
+    const [errMsg, setErrMsg] = useState<object | null>(null);
+
+    const scannedQR = (e: BarCodeReadEvent) => {
+        if(e?.data){
+            setShowModal(false);
+            setIsLoading(true)
+            const qrRawData = e?.data || ""
+            //Dynamic "00020201021226420014A000000615000101065891700210947061158552040000530345854031015802MY5908TESTNXAM6002MY63044CC4"
+            //Static "00020201021126420014A000000615000101066033460210MD000091515204546253034585802MY5916Kina's Bakehouse6002MY62730325167048055723900450702933405201678270121715006594007161670480304074001630474FA"
```
Comment (hiteshdigit101) [inline src/components/qr-transfer-component/scan-to-pay-component/index.tsx:41]: Please remove unnecessary comments and code

**Review by hiteshdigit101 — APPROVED** [review]

**Pull request merged**